pom.xml

```diff
@@ -62,4 +62,12 @@
         </profile>
 
     </profiles>
+
+     <dependencies>
+        <dependency>
+            <groupId>org.technologybrewery.habushu</groupId>
+            <artifactId>habushu-maven-plugin</artifactId>
+            <version>2.10.1-SNAPSHOT</version>
```

Comment:
Yes, the snapshot is the only version that has the functionality needed at this time. This will be updated in the future to a release version when it's published.